### ISSUE TITLE
Sometimes, new line in block code are not displayed

### ISSUE THREAD

**Issue by metal3d** [thorstenb/odpdown#37]
Nice work, really. I use odpdown with my project named vymad: I use VYM (mindmap tool) to manage my headings and content, then it converts the file to markdown that is treated with odpdown. But I have a strange problem.

Example: I have that content

``````
Some text

```bash
at 10:00 AM
at 9:00 PM December 21 2016
at now + 2min
```
``````

And in the odp file, I see:

![image](https://cloud.githubusercontent.com/assets/100080/12932344/a169c028-cf82-11e5-851c-692cb46b7230.png)

I don't find how to fix it... any idea ?

**Comment by thorstenb** [comment]
Good catch, there's one whitespace/lineend case missing. I'll get to that this week.

**Comment by metal3d** [comment]
No problem... That's just complicated, at this time, to manually fix more than 80 slides :(

Because I need it very quickly, I can try to change code (I'm python developper), but I don't know where I should start :p

**Comment by thorstenb** [comment]
if you don't absolutely need bash highlighter, just using

```
at 10:00 AM
at 9:00 PM December 21 2016
at now + 2min
```

works for me

**Comment by thorstenb** [comment]
And see commit above for a hacky, untested, etc WIP fix ;)

**Comment by metal3d** [comment]
Hi, excuse me for late answer, I was very busy.
I tried the last version (git clone) - the problem remains...
Removing "bash" highlight is ok... but I need it for other slides.
I'll check how to fix it

**Comment by metal3d** [comment]
See the screencap:
![image](https://cloud.githubusercontent.com/assets/100080/13048738/45aaa198-d3e8-11e5-90f5-02fdf7259546.png)

The problem remains on last line

**Comment by thorstenb** [comment]
Re-tested with v0.5.0 seems to work fine now.
